[2024-06-30T05:02:13.361+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-30T05:02:13.378+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: model_training_v4.upload_task scheduled__2024-06-29T00:00:00+00:00 [queued]>
[2024-06-30T05:02:13.383+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: model_training_v4.upload_task scheduled__2024-06-29T00:00:00+00:00 [queued]>
[2024-06-30T05:02:13.384+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-30T05:02:13.394+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_task> on 2024-06-29 00:00:00+00:00
[2024-06-30T05:02:13.399+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=10388) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-30T05:02:13.400+0000] {standard_task_runner.py:63} INFO - Started process 10390 to run task
[2024-06-30T05:02:13.400+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'model_training_v4', 'upload_task', 'scheduled__2024-06-29T00:00:00+00:00', '--job-id', '497', '--raw', '--subdir', 'DAGS_FOLDER/model_training.py', '--cfg-path', '/tmp/tmp378elmuh']
[2024-06-30T05:02:13.402+0000] {standard_task_runner.py:91} INFO - Job 497: Subtask upload_task
[2024-06-30T05:02:13.434+0000] {task_command.py:426} INFO - Running <TaskInstance: model_training_v4.upload_task scheduled__2024-06-29T00:00:00+00:00 [running]> on host 36c0809852d0
[2024-06-30T05:02:13.488+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='model_training_v4' AIRFLOW_CTX_TASK_ID='upload_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-29T00:00:00+00:00'
[2024-06-30T05:02:13.489+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-30T05:02:13.502+0000] {base.py:84} INFO - Using connection ID 'yandex_storage_conn' for task execution.
[2024-06-30T05:02:15.361+0000] {logging_mixin.py:188} INFO - File uploaded successfully to Yandex Object Storage
[2024-06-30T05:02:15.362+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-30T05:02:15.362+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-30T05:02:15.370+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=model_training_v4, task_id=upload_task, run_id=scheduled__2024-06-29T00:00:00+00:00, execution_date=20240629T000000, start_date=20240630T050213, end_date=20240630T050215
[2024-06-30T05:02:15.419+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-30T05:02:15.430+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-30T05:02:15.433+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
